galaxy.yml

```diff
@@ -1,6 +1,6 @@
 namespace: opentermsarchive
 name: deployment
-version: 0.0.5
+version: 0.0.6
```

Comment:
Galaxy release is already [0.0.6](https://galaxy.ansible.com/opentermsarchive/deployment)

Reply:
Yep, but this galaxy file is not up to date because the release was done from my computer. When this PR will be merged, the CI will automatically change this file and the changelog to version 0.0.7